components/konflux-ui/staging/base/external-secret.yaml

```diff
@@ -0,0 +1,28 @@
+---
+apiVersion: external-secrets.io/v1beta1
+kind: ExternalSecret
+metadata:
+  name: watson-config
+  namespace: konflux-ui
```

Comment:
you would need to add the `konflux-ui` namespace to the allow list in the ClusterSecretStore https://github.com/redhat-appstudio/infra-deployments/blob/main/components/cluster-secret-store/base/appsre-stonesoup-vault-secret-store.yaml#L29